### PR TITLE
EDU-3019: Clarifies confusion between pem and crt in Cloud Worker

### PR DESCRIPTION
**Pull request by fairlydurable**
Java, Go, Python, Typescript

Ticket source: @yuandrew

### CODE REVIEW

File: docs/getting_started/go/run_workers_with_cloud_go/index.md [hunk 1]
```diff
@@ -215,8 +224,8 @@ Before running your application, set the following environment variables with th
   <TabItem value="macos" label="MacOS">
 
 ```bash
-export TEMPORAL_MTLS_TLS_CERT=/path/to/tls/cert.pem
-export TEMPORAL_MTLS_TLS_KEY=/path/to/tls/key.pem
+export TEMPORAL_MTLS_TLS_CERT=C:\path\to\tls\cert-file.(pem|crt)
```
Comment (yuandrew) [inline docs/getting_started/go/run_workers_with_cloud_go/index.md:227]: This set of code is for MacOS, original path should be maintained

**Review by yuandrew — APPROVED** [review]
Changes look good, besides the one path issue! Once that's fixed, should be good to merge

File: docs/getting_started/go/run_workers_with_cloud_go/index.md (hunk 1 above)
Comment (fairlydurable) [reply docs/getting_started/go/run_workers_with_cloud_go/index.md:227]: Fixed.

Comment (yuandrew) [reply docs/getting_started/go/run_workers_with_cloud_go/index.md:227]: This still seems to be windows paths, did you perhaps forget to push your changes?

Comment (fairlydurable) [reply docs/getting_started/go/run_workers_with_cloud_go/index.md:227]: Sorry about that. Was trying to decide with Brian whether to finish this as a single ticket or do all SDKs as separate tickets. Changes are still local on my system.

File: docs/getting_started/go/run_workers_with_cloud_go/index.md [hunk 2]
```diff
@@ -40,11 +40,20 @@ They are required for configuring mutual Transport Layer Security (mTLS) protoco
 You have a few options to generate certificates.
 
 - Use existing certificate management infrastructure to generate certificates for your Namespace.
-- Use Temporal's built-in certificate generation tool [tcld](https://docs.temporal.io/cloud/tcld).
-- Use open-source tools like [certstrap](https://github.com/square/certstrap).
+- Use Temporal's built-in certificate generation tool [`tcld`](https://docs.temporal.io/cloud/tcld) to create a `.pem` file containing the certificate.
+- Use open-source tools like [Certstrap](https://github.com/square/certstrap).
+  The sample below creates a `.crt` certificate file with Certstrap, but the tool can also output `.pem` format like `tcld`.
```
Comment (napcs) [inline docs/getting_started/go/run_workers_with_cloud_go/index.md:45]: Let's avoid "below" and make it more active voice.  The sentence that follows says a similar thing too - that you'll use certstrap.

**Review by napcs — DISMISSED** [review]

File: docs/getting_started/go/run_workers_with_cloud_go/index.md (hunk 2 above)
Comment (fairlydurable) [reply docs/getting_started/go/run_workers_with_cloud_go/index.md:45]: Will fix.

**Review by napcs — APPROVED** [review]

**Pull request merged**